cli/cmd/generate.go

```diff
@@ -142,6 +145,28 @@ var (
 				return errors.Wrap(err, "failed to load saved options!")
 			}
 
+			// Parse passed in subaccounts (if any)
+			if GenerateAwsCommandExtraState.AwsSubAccounts != "" {
+				// validate consolidated_cloudtrail is enabled - otherwise this flag cannot be used
+				if ok, _ := cmd.Flags().GetBool("consolidated_cloudtrail"); !ok {
+					return errors.New("aws subaccounts can only be supplied with consolidated cloudtrail enabled")
+				}
+
+				// validate the format of supplied value is correct
+				matchRegEx := `(\w+:\w+(,)?)+`
```

Comment:
Similar to the arn regex it would be nice to see a unit test for this.

Reply:
👍